src/resources/assets/js/components/dismissImageGridImage.vue

```diff
@@ -41,6 +64,32 @@ export default {
         title() {
             return this.selected ? 'Undo dismissing this annotation' : 'Dismiss this annotation';
         },
+        isPoint() {
+            return this.svg !== null ? this.svg.includes('r="3"') : false;
+        },
+        svgSrcUrl() {
+            // Replace file extension by svg file format
+            return this.srcUrl.replace(/.[A-Za-z]*$/, '.svg');
+        },
+    },
+    methods: {
+        async fetchSVG() {
+            this.label_id = this.image.label_id;
+            let response = await fetch(this.svgSrcUrl);
+
+            if (!response.ok) {
+                return;
+            }
+
+            this.svg = await response.text();
+        },
+        getSVGStyle(isOutline) {
+            return isOutline ? '--color: white; --width: 5px;'
+                : '--color: #' + this.image.label_color + '; --width: 3px;';
```

Comment:
Nice use of CSS variables, I haven't thought of using them this way until now 👍

String concatenation like this can also be easier with JS template strings:

```js
`--color: #${this.image.label_color}; --width: 3px;`
```